mut/redirects/redirect_main.py

```diff
@@ -0,0 +1,396 @@
+"""
+Usage:
+    mut-redirects <source_path> -o <output>
+
+    -h, --help             List CLI prototype, arguments, and options.
+    <source_path>          Path to the file(s) containing redirect rules.
+    -o, --output <output>  File path for the output .htaccess file.
+"""
+
+# Spec URL: https://docs.google.com/document/d/1oI2boFmtzvbbvt-uQawY9k_gLSLbW7LQO2RjVkvtRgg/edit?ts=57caf48b#
+
+import re
+from docopt import docopt
+
+rules = []
+symlinks = []
+definitions = []
+
+class Rule:
+    is_temp = None
```

Comment:
These attributes should be *instance* attributes rather than *class* attributes.

But I might actually use a named tuple rather than a class for performance reasons:
```python
import collections
RuleDefinition = collections.namedtuple('RuleDefinition', ('is_temp', 'version', 'old_url', 'new_url'))
```